thesis-style/thesis.tex

```diff
@@ -24,6 +24,15 @@
 
 \usepackage{hyperref}
 
+% Ensure that urls longer than the page width are broken up
```

Comment:
Can you reference the source here (author, url)? Thanks!

Reply:
Done :+1: